### ISSUE TITLE
create-app-manifest wraps an extra set of quotes around environment variables

### ISSUE THREAD

**Issue by flavorjones** [cloudfoundry/cli#800]
### Command

`cf create-app-manifest` 

### CLI Version

```
cf version 6.16.1+924508c-2016-02-26
```

### Error

Repeatedly running `cf create-app-manifest` and `cf push` will cause environment variables to get wrapped in extra pairs of double-quotes (`"`).

If I start with `set env appname FOO bar`, then `cf env` will emit:

```
User-Provided:
FOO: bar
```

but `cf create-app-manifest` will write the following:

```
  env:
    FOO: '"bar"'
```

If I push this manifest, then `cf env` will emit:

```
User-Provided:
FOO: "bar"
```

and another run of `cf create-app-manifest` will write:

```
  env:
    FOO: '""bar""'
```

I'd expect this series of operations to be idempotent, and for the environment variable to always be the same string, `bar`, without quotes.

### Stack Trace

n/a

### CC API Version

```
$ cf api
API endpoint: https://api.run.pivotal.io (API version: 2.52.0)
```

### CF Trace

For bugs related to HTTP requests, please run the command with env var `CF_TRACE=true`

### Platform Details
```
$ uname -a
Linux curiosity 3.19.0-32-generic #37~14.04.1-Ubuntu SMP Thu Oct 22 09:41:40 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux
```

### Shell

terminator

**Comment by dkoper** [comment]
Thanks Mike. It was easy to reproduce; the team will investigate.

**Comment by dkoper** [comment]
This has been fixed in cf CLI v6.17.0.
https://github.com/cloudfoundry/cli#downloads

Thanks again!
Dies Koper
CF CLI PM
